orchestration/dap_orchestration/solids.py

```diff
@@ -241,8 +242,59 @@ def write_outfiles(context: AbstractComputeExecutionContext, fan_in_results: lis
     NOTE: The fan_in_results param allows to introduce a fan-in dependency from upstream transformation
     solids, but is ignored by this solid.
     """
+    tsv_directory = context.solid_config["output_dir"]
     context.resources.outfiles_writer.run(
-        context.solid_config["output_dir"],
+        tsv_directory,
         context.resources.refresh_directory,
         fan_in_results
     )
+
+    return tuple((fan_in_results, tsv_directory))
```

Comment:
I had trouble just trying to return "fan_in_results, tsv_directory"
as it kept making my pipeline tests fail with the message:
E           dagster.core.errors.DagsterInvalidConfigError: Error in config for pipeline refresh_data_all
E               Error 1: Missing required config entry "inputs" at path root:solids:upload_to_gcs. Sample config for missing entry: {'inputs': {'tsv_directory': '...'}}
But I wouldn't see the same error for the "fan_in_results" argument. And the scheduled_sample imported from prod_repositories also always passed without issues.